dynawo/sources/Models/Modelica/PreassembledModels/DiscreteIntegrator.xml

```diff
@@ -12,7 +12,7 @@
     simulation tools for power systems.
 -->
 <dyn:dynamicModelsArchitecture xmlns:dyn="http://www.rte-france.com/dynawo">
-  <dyn:modelicaModel id="DiscretePI">
-    <dyn:unitDynamicModel id="pI" name="Dynawo.NonElectrical.Blocks.Continuous.DiscretePI"/>
+  <dyn:modelicaModel id="DiscreteIntegrator">
+    <dyn:unitDynamicModel id="DiscreteIntegrator" name="Dynawo.NonElectrical.Blocks.Continuous.DiscreteIntegrator"/>
```

Comment:
I am not sure we are allowed to use the same name in the id here as the is above and the name. I suggest discretePI